src/app/pages/progress-report/components/progress-report-chart/chart-styles.ts

```diff
@@ -0,0 +1,13 @@
+export const enum markerColors {
```

Comment:
Can we move this file somewhere in `/src/app/shared/` or `/src/app/shared/enums/`?

Reply:
Yes, I just wanted to keep all the files related to this module inside it. But it is possible to merge it with the shared module.